pkg/api/openshiftcluster.go

```diff
@@ -204,4 +204,5 @@ const (
 	InstallPhaseDeployStorage InstallPhase = iota
 	InstallPhaseDeployResources
 	InstallPhaseRemoveBootstrap
+	InstallPhaseConfigureCluster
```

Comment:
FYI https://github.com/Azure/ARO-RP/issues/106

Reply:
I think we shoud have more, not less. Just because they take ages and it easier to handle restarts

Reply:
I think @jim-minter was of the opinion of less is better. I am in the process of refactoring this code to have 2 steps.